### ISSUE TITLE
App not compiling with kotlin_version 1.8.21

### ISSUE THREAD

**Issue by CodeWizard0367** [hui-z/image_gallery_saver#283]
* What went wrong:
Execution failed for task ':image_gallery_saver:compileDebugKotlin'.
> 'compileDebugJavaWithJavac' task (current target is 1.8) and 'compileDebugKotlin' task (current target is 17) jvm target compatibility should be set to the same Java version.
  Consider using JVM toolchain: https://kotl.in/gradle/jvm/toolchain
2

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 1m 33s
Exception: Gradle task assembleDebug failed with exit code 1

**Comment by krutarth-vaishnav** [comment]
Yes, I'm facing the same issue. It's open since last week, Still no response, Hello?

**Comment by knottx** [comment]
https://github.com/hui-z/image_gallery_saver/pull/280

**Comment by knottx** [comment]
@CodeWizard0367  @krutarth-vaishnav  
try this #295
